style/sass/README.md

```diff
@@ -27,29 +27,178 @@
 * Use a `%` unit for the amount/weight when using Sass's color functions: `darken($color, 20%)`, not `darken($color, 20)`
 * Use a trailing comma after each item in a map, including the last item.
 
-## Order
-
-* Use alphabetical order for declarations.
-* Place `@extends` and `@includes` at the top of your declaration list.
-* Place media queries directly after the declaration list.
-* Place pseudo-classes and pseudo-elements second.
-* Place nested elements third.
-* Place nested classes fourth.
-
 ## Selectors
 
-* Don't use ID's for style.
-* Avoid over-qualified selectors: `h1.page-title`, `div > .page-title`
 * Use meaningful names: `$visual-grid-color` not `$color` or `$vslgrd-clr`.
-* Be consistent about naming conventions for classes. For instance, if a project is using BEM, continue using it, and if it's not, do not introduce it.
 * Use ID and class names that are as short as possible but as long as necessary.
 * Avoid nesting more than 3 selectors deep.
 * Avoid using comma delimited selectors.
 * Avoid nesting within a media query.
-* Don't concatenate selectors using Sass's parent selector (`&`).
 
 ## Organization
 
 * Use a `base` directory for styling element selectors, global variables, global extends and global mixins.
 * Use HTML structure for ordering of selectors. Don't just put styles at the bottom of the Sass file.
 * Avoid having files longer than 100 lines.
+
+---
+
+## Selectors
+
+### Selector specificity
+
+- Don't use ID selectors.
+- Avoid over-qualified selectors, e.g. `h1.page-title`.
+
+<details>
+
+#### Code examples
+
+`h1.page-title` carries a specificity of 2, but can be reduced to 1 by removing
+the `h1` type selector:
+
+```diff
+-h1.page-title
++.page-title {
+   // …
+ }
+```
+
+#### Motivation
+
+Using an ID in a selector increases its specificity, making it more difficult
+to work with alongside class selectors. Furthermore, because IDs must be unique
+within an HTML document, using them as CSS selectors limits reusability.
+
+#### Resources
+
+- Learn about [how specificity is calculated][specificity-calculation].
```

Comment:
I really appreciate sharing these resources. If these were available for each section that would be so cool!

Reply:
I definitely intend to keep adding more!